optimize/starter/README.md

```diff
@@ -0,0 +1,165 @@
+# Prisma Optimize example
+
+This repository demonstrates how to setup and use [Prisma Optimize](https://pris.ly/optimize).
+
+## Prerequisites
+
+To successfully run the project, you will need the following:
+
+1. The **database connection string** that is supported by Prisma Optimize.
+2. An active [Prisma Data Platform](https://pris.ly/pdp) account.
+
+## Getting started
+
+### 1. Clone the respository
+
+Clone the repository, navigate into it and install dependencies:
+
+```terminal
+git clone git@github.com:prisma/prisma-examples.git --depth=1
+cd prisma-examples/optimize/starter
+npm install
+```
+
+### 2. Configure environment variables
+
+Create a `.env` in the root of the project directory:
+
+```terminal
+touch .env
+```
+
+Now, open the `.env` file and update the `DATABASE_URL` environment variable with the value of your connection string:
+
+```env
+# .env
+DATABASE_URL="__YOUR_DATABASE_CONNECTION_STRING__"
+# Note that __YOUR_DATABASE_CONNECTION_STRING__  is a placeholder value that you need to replace with the values of your connection string.
+```
+
+## 3. Setup the project
+
+Perform a database migration for the project to work:
+
+```terminal
+npx prisma migrate dev --name init
+```
+
+Then, fill your database with some values by running the [seed script](./prisma/seed.ts):
+
+```terminal
+npx prisma db seed
+```
```

Comment:
this should not be necessary since the initial migration automatically executes the seed script.

```suggestion
```